addons/account/models/account_move.py

```diff
@@ -27,7 +27,8 @@
     frozendict,
     get_lang,
     is_html_empty,
-    sql
+    sql,
+    groupby
```

Comment:
Should be ordered alphabetically (according to the guidelines https://www.odoo.com/documentation/master/contributing/development/coding_guidelines.html#imports )